native/SampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -292,6 +297,7 @@ private void handleSyncUpdate(SyncState sync) {
 				Toast.makeText(MainActivity.this,
 						"Sync down successful!",
 						Toast.LENGTH_LONG).show();
+                refreshList();
```

Comment:
Without the refresh call, the list would remain empty until the app was paused/resumed.